src/Widgets/BridgeWidgetApi.ts

```diff
@@ -156,7 +156,7 @@ export class BridgeWidgetApi {
         if (!connection.provisionerUpdateConfig || !connection.getProvisionerDetails)  {
             throw new ApiError("Connection type does not support updates", ErrCode.UnsupportedOperation);
         }
-        await connection.provisionerUpdateConfig(req.userId, req.body);
+        connection.provisionerUpdateConfig(req.userId, req.body);
```

Comment:
Did this stop being promisey?

Reply:
Yes: https://github.com/matrix-org/matrix-hookshot/blob/main/src/Connections/IConnection.ts#L68